<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#loginPage,
QWidget#modeSelectionPage,
QStackedWidget {
    background-image: url(:/img/img/HomePage.png);
    background-repeat: no-repeat;
    background-position: center;
}

/* Text input fields */
QLineEdit {
    background-color: #444444;
    color: white;
    border: 1px solid #888888;
    padding: 4px;
}

/* Labels */
QLabel {
    color: white;
    background-color: #444444;
    padding: 2px;
}

/* Buttons */
QPushButton {
    background-color: #555555;
    color: white;
    border: 1px solid #888888;
    padding: 6px;
    border-radius: 4px;
}

QPushButton:hover {
    background-color: #666666;
}

QPushButton:pressed {
    background-color: #333333;
}

/* Radio Buttons */
QRadioButton {
    color: white;
    background-color: #444444;
    border: 1px solid #888888;
    padding: 4px;
    border-radius: 3px;
border-image: url(:/img/img/gray.jpg) 0 0 0 0 stretch stretch;

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">QWidget {
	border-image: url(:/img/img/HomePage.png) 0 0 0 0 stretch stretch;
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover;
}
QLineEdit {
    border-image: url(:/img/img/gray.jpg) 0 0 0 0 stretch stretch;
    color: white;
    border: 1px solid #888888;
    padding: 4px;
}

QLabel {
    border-image: url(:/img/img/gray.jpg) 0 0 0 0 stretch stretch;
    color: white;
}

QPushButton {
    background-color: #555555;
    color: white;
    border: 1px solid #888888;
    padding: 6px;
    border-radius: 4px;
    border-image: url(:/img/img/gray.jpg) 0 0 0 0 stretch stretch;

}

QPushButton:hover {
    background-color: #666666;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="usernameEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>&quot;Username&quot;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="passwordEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Password</enum>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>&quot;Password&quot;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="signInButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sign In</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="signUpButton">
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    background-color: #444444;
    color: white;
    border: 1px solid #888888;
    padding: 4px;
}

QLabel {
    color: white;
}

QPushButton {
    background-color: #555555;
    color: white;
    border: 1px solid #888888;
    padding: 6px;
    border-radius: 4px;
}

QPushButton:hover {
    background-color: #666666;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
            </property>
            <property name="text">
             <string>Sign Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="guestButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Play as Guest</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modeSelectionPage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="6" column="3">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="3">
           <widget class="QComboBox" name="aiDifficultyCombo">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
    background-color: #444444;
    color: white;
    border: 1px solid #888888;
    padding: 4px;
    border-radius: 4px;
}

QComboBox QAbstractItemView {
    background-color: #444444;
    color: white;
    selection-background-color: #666666;
    selection-color: white;
    border: 1px solid #888888;
}
</string>
            </property>
            <property name="currentText">
             <string>Easy</string>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Easy</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/img/img/easy.jpg</normaloff>:/img/img/easy.jpg</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/img/img/medium.jpg</normaloff>:/img/img/medium.jpg</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hard</string>
             </property>
             <property name="icon">
              <iconset>
               <disabledoff>:/img/img/hard.jpg</disabledoff>
              </iconset>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Choose Game Mood</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="vsPlayerRadio">
            <property name="text">
             <string>Player vs Player</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QRadioButton" name="vsAIRadio">
            <property name="text">
             <string>Player vs AI</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="startGameButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubarMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actionGameHistory">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Game History</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
